core/src/main/resources/error/error-classes.json

```diff
@@ -104,6 +172,10 @@
     "message" : [ "The target JDBC server does not support transaction and can only support ALTER TABLE with a single action." ],
     "sqlState" : "0A000"
   },
+  "WINDOW_NOT_SUPPORT_MERGING" : {
```

Comment:
This doesn't make sense grammatically; can we rephrase it along the lines of "MERGE_UNSUPPORTED_BY_WINDOW_FUNCTIONS"?